deploy/olm-catalog/ibm-iam-operator/3.18.0/ibm-iam-operator.v3.18.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,2046 @@
+          "metadata": {
+            "name": "ibm-iam-bindinfo"
+          },
+          "spec": {
+            "operand": "ibm-iam-operator",
+            "registry": "common-service",
+            "description": "Binding information that should be accessible to iam adopters",
+            "bindings": {
+              "public-oidc-creds": {
+                "secret": "platform-oidc-credentials"
+              },
+              "public-auth-creds": {
+                "secret" : "platform-auth-idp-credentials"
+              },
+              "public-auth-cert": {
+                "secret" : "platform-auth-secret"
+              },
+              "public-cam-secret": {
+                "secret" : "oauth-client-secret"
+              },
+              "protected-zen-serviceid": {
+                "secret" : "zen-serviceid-apikey-secret"
+              },
+              "public-cam-map": {
+                "configmap" : "oauth-client-map"
+              },
+              "public-auth-config": {
+                "configmap" : "platform-auth-idp"
+              }
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Security
+    certified: "false"
+    containerImage: quay.io/opencloudio/ibm-iam-operator:latest
+    description: The IAM operator provides a simple Kubernetes CRD-Based API to manage
+      the lifecycle of IAM services. With this operator, you can simply deploy and
+      upgrade the IAM services
+    olm.skipRange: '<3.17.0'
+    repository: https://github.com/IBM/ibm-iam-operator
+    support: IBM
+  labels:
+    operatorframework.io/arch.s390x: supported
+    operatorframework.io/os.linux: supported
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.ppc64le: supported
+  name: ibm-iam-operator.v3.17.0
+  namespace: placeholder
+spec:
+  relatedImages:
+    - name: IBM_IAM_OPERATOR_IMAGE
+      image: quay.io/opencloudio/ibm-iam-operator:3.17.0
+    - name: ICP_PLATFORM_AUTH_IMAGE
+      image: quay.io/opencloudio/icp-platform-auth:3.7.6
+    - name: ICP_IDENTITY_PROVIDER_IMAGE
+      image: quay.io/opencloudio/icp-identity-provider:3.7.6
+    - name: ICP_IDENTITY_MANAGER_IMAGE
+      image: quay.io/opencloudio/icp-identity-manager:3.7.6
+    - name: IAM_POLICY_ADMINISTRATION_IMAGE
+      image: quay.io/opencloudio/iam-policy-administration:3.7.6
+    - name: IAM_POLICY_DECISION_IMAGE
+      image: quay.io/opencloudio/iam-policy-decision:3.7.6
+    - name: ICP_SECRET_WATCHER_IMAGE
+      image: quay.io/opencloudio/icp-secret-watcher:3.7.6
+    - name: ICP_OIDCCLIENT_WATCHER_IMAGE
+      image: quay.io/opencloudio/icp-oidcclient-watcher:3.7.6
+    - name: IAM_POLICY_CONTROLLER_IMAGE
+      image: quay.io/opencloudio/iam-policy-controller:3.7.6
+    - name: ICP_IAM_ONBOARDING_IMAGE
+      image: quay.io/opencloudio/icp-iam-onboarding:3.7.6
+    - name: AUDIT_SYSLOG_SERVICE_IMAGE
+      image: quay.io/opencloudio/audit-syslog-service:1.7.0
```

Comment:
Please change to audit-syslog-service:1.8.0 (version delivered with 3.17)
Do this change other places also.

Reply:
done